mysql/README.md

```diff
@@ -0,0 +1,233 @@
+## MySQL Example Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+To use HarbourBridge on a MySQL database called mydb using mysqldump, run:
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump
+```
+
+The tool can also be applied to an existing mysqldump file:
+
+```sh
+harbourbridge -driver=mysqldump < my_mysqldump_file
+```
+
+To specify a particular Spanner instance to use, run:
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump -instance my-spanner-instance
+```
+
+By default, HarbourBridge will generate a new Spanner database name to populate.
+You can override this and specify the database name to use by:
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump -dbname my-spanner-database-name
+```
+
+HarbourBridge generates a report file, a schema file, and a bad-data file (if
+there are bad-data rows). You can control where these files are written by
+specifying a file prefix. For example,
+
+```sh
+mysqldump mydb | harbourbridge -prefix mydb. -driver=mysqldump
+```
+
+will write files `mydb.report.txt`, `mydb.schema.txt`, and
+`mydb.dropped.txt`. The prefix can also be a directory. For example,
+
+```sh
+mysqldump mydb | harbourbridge -prefix ~/spanner-eval-mydb/ -driver=mysqldump
+```
+
+would write the files into the directory `~/spanner-eval-mydb/`. Note
+that HarbourBridge will not create directories as it writes these files.
+
+To use the tool directly on a MySQL database called mydb, run
+
+```sh
+harbourbridge -driver=mysql
+```
+
+It is assumed that MySQL database supports information schema tables and
+_MYSQLHOST_, _MYSQLPORT_, _MYSQLUSER_, _MYSQLDATABASE_ environments variables are set.
+Password can be specified either in the _MYSQL_PWD_ environment variable or need to
+enter password when prompted by harbourbridge.
```

Comment:
Suggest replacing:
"or need to enter password when prompted by harbourbridge."
by
"or provided at the password prompt."